src/misc.h

```diff
@@ -190,14 +190,16 @@ const char *jtr_lltoa(int64_t num, char *result, int result_len, int base);
 const char *jtr_ulltoa(uint64_t num, char *result, int result_len, int base);
 
 /*
- * Change some large number to a string possibly using SI prefix
- * eg. 437281954 -> "417 M"
+ * Change some large number to a string possibly using binary prefix,
+ * eg. 437281954 -> "417 Mi"
+ * Note: "leaks" 16 bytes each time (mem_alloc_tiny)
  */
 extern char *human_prefix(uint64_t num);
 
 /*
  * Change some tiny number to a string possibly using SI prefix
  * eg. 0.123 -> "123 m"
+ * Note: "leaks" 16 bytes each time (mem_alloc_tiny)
```

Comment:
The problem with leaks (not sure if this will be one problem) is CI.

CI is the monkey we have to run a fast/dumb (and very useful) checks. If we insert acceptable leaks, we can no longer use the helpful monkey. I suggest we should avoid it by design (I mean, fix it if it is a problem).

Reply:
Right. Yet I can see how it'd be nasty to free this memory after every call to these, or to have them use one static buffer, as they might be called multiple times from arguments to one `printf()` or such. A crazy way I've seen how people workaround this in C is have functions like these iterate over several static buffers, like:

```c
static unsigned int bufid;
static char bufs[8][16];
char *buf = bufs[bufid++ & 7];
```

but then the user of such functions needs to remember that while a limited number of them is OK to use in one `printf()`, their returned pointers are nevertheless not OK to store long-term. I have mixed feelings about this.

Reply:
> but then the user of such functions needs to remember that while a limited number of them is OK to use in one `printf()`, their returned pointers are nevertheless not OK to store long-term. I have mixed feelings about this.

I'm guilty of doing that, maybe even in this repo. I think this tiny alloc is better (for this very use).

It's not actually a memory leak by most definitions as there's nowhere this is used in any loop and/or over time - it's merely that we never free it until just before exit (Jumbo-specific, to help with leak debugging), just like lots of other stuff. Anyway while touching the comments I thought I'd add a note about it.